examples/deck.gl/3d-tiles/app.js

```diff
@@ -24,7 +24,7 @@ import {INITIAL_MAP_STYLE} from './constants';
 // enable DracoWorkerLoader when fixed
 registerLoaders([GLTFLoader, DracoLoader]);
 setLoaderOptions({
-  worker: 'local'
+  worker: true
```

Comment:
This is the default, right? I think we can delete the `setLoaderOptions` call then.